topics/proteomics/tutorials/ml-modeling-of-anti-cancer-peptides/tutorial.md

```diff
@@ -0,0 +1,284 @@
+---
+layout: tutorial_hands_on
+
+title: ML Modeling of Anti-cancer Peptides
+questions:
+  - Which ML algorithm is superior in predicting anti-cancer peptides?
+objectives:
+  - Learn, how to calculate peptide descriptor
+  - Learn, how to create training data set from features?
+  - Assessment of best ML algorithm in predicting anti-cancer peptide
+time_estimation: '30m'
+
+contributors:
+  - jaidevjoshi83
+  - blankenberg
+---
+
+## Introduction
+{:.no_toc}
+
+Several computational methods have been proven very useful in the initial screening and prediction of peptides for various biological properties. These methods have emerged as effective alternatives to the lengthy and expensive traditional experimental approaches. In this tutorial, we will be discussing how peptide-based properties like charge, hydrophobicity, the composition of amino acids, etc. can be utilized to predict the biological properties of peptides. In this tutorial, we will learn how to use different utilities of PDAUG to calculate various peptide-based descriptors and use these descriptors for machine learning modeling of peptides with known anti-cancer properties. We will use CTD (composition, transition, and distribution ) descriptor to define peptide sequences in the training set and will test 6 different machine learning algorithms. We will also assess the effect of normalization on the accuracy of ML models.
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+## Training data set
+
+A high-quality dataset was retrieved from a previously published work (Hajisharifi et al., 2014). A balanced dataset, 138 ACPs (positive), and 138 non-ACPs (negative) sequences were obtained by randomly removing negative sequences. The length distribution of the positive dataset is somewhat different from the negative dataset.
```

Comment:
Could you use a citation here? https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#citations

Reply:
Citation added